yellowbrick/features/importances.py

```diff
@@ -185,7 +190,25 @@ def draw(self, **kwargs):
         pos = np.arange(self.features_.shape[0]) + 0.5
 
         # Plot the bar chart
-        self.ax.barh(pos, self.feature_importances_, align='center')
+        if self.stack:
+            colors = color_palette(kwargs.pop('colors', None),
+                                   len(self.classes_))
+            zeros = np.zeros(self.feature_importances_.shape[1])
+            left_arr = np.zeros((self.feature_importances_.shape[1], 2))
+
+            for idx in range(len(self.feature_importances_)):
+                left = [left_arr[j, int(self.feature_importances_[idx][j]>0)]
+                        for j in range(len(self.feature_importances_[idx]))]
+
+                self.ax.barh(pos, self.feature_importances_[idx], left=left,
+                             color=colors[idx], label=self.classes_[idx])
+
+                left_arr[:, 0] += np.minimum(self.feature_importances_[idx],
+                                             zeros)
+                left_arr[:, 1] += np.maximum(self.feature_importances_[idx],
+                                             zeros)
```

Comment:
Very good implementation for the stacked bar chart! I always find the matplotlib stacked charts tricky to implement; we probably should create a helper function do this if we're going to have other stacked bar charts in Yellowbrick! Would you mind creating an issue for this if you think that's a good idea?

Reply:
+1